src/Data/Swagger/Lens.hs

```diff
@@ -62,17 +62,17 @@ makePrisms ''Referenced
 
 _SwaggerItemsArray :: Review (SwaggerItems 'SwaggerKindSchema) [Referenced Schema]
 _SwaggerItemsArray
-  = prism (\x -> SwaggerItemsArray x) $ \x -> case x of
+  = unto (\x -> SwaggerItemsArray x) {- $ \x -> case x of
```

Comment:
@fizruk I'm wasn't sure, if this and `_SwaggerItemsObject ` are lawful `Prism'`, so changed to `unto`.